catalog/ui/catalog-ui-search/package.json

```diff
@@ -5,11 +5,13 @@
   "version": "0.0.1",
   "license": "LGPL-3.0",
   "scripts": {
-    "startplus": "grunt startplus --webpackConfig=./webpack/config/dev.js",
```

Comment:
Having some build issues when I remove some of these, but not sure why.  The maven bundle plugin is failing.